pkg/manifests/manifests.go

```diff
@@ -2558,7 +2558,9 @@ func (f *Factory) NewService(manifest []byte) (*v1.Service, error) {
 	return &s, nil
 }
 
+//nolint:staticcheck // MON-4216 to track migration from Endpoints to EndpointSlice
```

Comment:
out of curiosity, what was staticcheck complaining about? the method not being used?